src/internal/components/pluggableVisualizations/pivotTable/tests/PluggablePivotTable.spec.tsx

```diff
@@ -356,6 +371,46 @@ describe("PluggablePivotTable", () => {
             expect(props.config).toEqual(extendedConfig);
         });
 
+        it("should have onColumnResized callback when FF enableTableColumnsManualResizing is set to true", () => {
+            const pivotTable = createComponent({
+                ...defaultProps,
+                featureFlags: { enableTableColumnsManualResizing: true },
+            });
+
+            const createElementSpy = spyOnFakeElement();
+            const renderSpy = spyOnRender();
+
+            const options = getDefaultOptions();
+            pivotTable.update(options, {}, testMocks.emptyMdObject);
+
+            expect(createElementSpy).toHaveBeenCalledTimes(1);
+            expect(createElementSpy.mock.calls[0][0]).toBe(PivotTable);
+
+            const props: any = createElementSpy.mock.calls[0][1];
+            expect(props.onColumnResized).toBeInstanceOf(Function);
+            spyOnCleanup(renderSpy, createElementSpy);
+        });
+
+        it("should have onColumnResized callback when FF enableTableColumnsManualResizing is set to false", () => {
```

Comment:
I am missing some No, Not, Undefined or something in description

Reply:
Done